Parsers/ASimFileEvents/ProductParsers/FileEventMicrosoftSharePoint.yaml

```diff
@@ -0,0 +1,69 @@
+Parser:
+  Title: Microsoft SharePoint File Operation - File Event Parser
+  Version: '0.1'
+  LastUpdated: June 28, 2021
+Product:
+  Name: Microsoft SharePoint
+Normalization:
+  Schema: FileEvent
+  Version: '0.1.0'
+References:
+- Title: ASIM File Event Schema
+  Link: https://aka.ms/AzSentinelFileEventDoc
+- Title: ASIM
+  Link: https://aka.ms/AzSentinelNormalization
+- Title: Office 365 Management Activity API schema
+  Link: https://docs.microsoft.comoffice/office-365-management-api/office-365-management-activity-api-schema#sharepoint-file-operations
+Description: |
+  This is a Query Parser that is used to map SharePoint File Operation (OfficeActivity) to the Azure Sentinel Information Model FileEvent schema.
+ParserName: vimFileEventMicrosoftSharePoint
```

Comment:
Getting rusty, so I might repeat. The same parser supports also OneDrive for business, so the texts should say that.

Reply:
added to title and description